control-plane-operator/hostedclusterconfigoperator/controllers/resources/registry/registry.go

```diff
@@ -26,7 +27,7 @@ func ReconcileRegistryConfig(cfg *imageregistryv1.Config, isPlatformNone bool) {
 	cfg.Spec.Requests.Write.MaxRunning = 0
 	cfg.Spec.Requests.Write.MaxWaitInQueue.Reset()
 
-	if isPlatformNone {
+	if platform == hyperv1.KubevirtPlatform || platform == hyperv1.NonePlatform {
```

Comment:
Why this change?
How is the change related to this PR? "Switching KubeVirt platform's default ingress to NodePort"
We should possibly run once and stop ReconcileRegistryConfig so user has the ability configure the registry not only for IBM, so unless there's a reason for this change to be here, I'd suggest to have a separate PR discussion.